packages/extension-chakra-storefront/src/components/link/index.jsx

```diff
@@ -10,23 +10,37 @@ import {Link as ChakraLink} from '@chakra-ui/react'
 import {Link as SPALink, NavLink as NavSPALink} from 'react-router-dom'
 import useMultiSite from '../../hooks/use-multi-site'
 
-const Link = React.forwardRef(({href, to, useNavLink = false, ...props}, ref) => {
+const Link = React.forwardRef(({href, to, useNavLink = false, css, children, ...props}, ref) => {
     const _href = to || href
     const {buildUrl} = useMultiSite()
     const updatedHref = buildUrl(_href)
+
+    const isActive = useNavLink
+        ? (_, location) => {
+              return location.pathname.endsWith(_href)
```

Comment:
Since we have multi-site, so we only need check the last section of the pathname.